README.md

```diff
@@ -116,7 +116,7 @@ total number of embedding in this folder
 
 #### .dimension
 
-dimension of one embedding
+dimension of one embedding (tuple of `(dim,)` for 1d embeddings or `(dim1, dim2)` for 2d embeddings)
```

Comment:
could you make it be a single number instead of a tuple for dimension one ? that way it wouldn't be a breaking change

Reply:
hmm that or we need to PR in all the users

Reply:
at least there https://github.com/criteo/autofaiss/search?q=dimension

Reply:
yeah let's make it a major version and update dependents